### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by thsutton**
Location, book, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,42 +16,26 @@ All are welcome!
...
-
-
-
 &#42; Programming Languages and Types (yet perfectly understandable) in Sydney &mdash; thanks to Danny Gratzer for the name suggestion.
 
 
 ## When and where
 
-We meet on Wednesday lunchtimes from 12 noon to 1:30.
+We meet on Wednesday lunchtimes from 12:30 to 2:00 pm.
```
Comment (jberthold) [inline README.md]: Monday?

**Review by jberthold — COMMENTED** [review]
...although we might as well go back to Wednesday ;-)

File: README.md (hunk 1 above)
Comment (thsutton) [reply README.md]: derp

**Comment by mjhopkins** [comment]
LGTM

**Pull request merged**